js/viz/gauges/utils/resolve_label_overlapping.js

```diff
@@ -0,0 +1,104 @@
+const ARC_COORD_PREC = 5;
+
+import { normalizeAngle, normalizeArcParams } from '../../core/utils';
+
+export function dividePoints(bars) {
+    return bars.reduce(function(r, bar) {
+        const angle = normalizeAngle(bar._angle);
+        bar._text._lastCoords = { x: 0, y: 0 };
+        (angle <= 90 || angle >= 270 ? r.right : r.left)
+            .push({
+                series: {
+                    isStackedSeries: ()=> false,
```

Comment:
Are lint rules different for charts' code? I think should look like this everywhere
```suggestion
                    isStackedSeries: () => false,
```

Reply:
fixed